File: demo/springboot/demo/src/main/java/com/demo/domain/dto/http/resonse/UserPasswordUpdateResponse.java
package com.demo.domain.dto.http.resonse;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

@Data
public class UserPasswordUpdateResponse {
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private Long userId;
    private Boolean updated;
    private String message;
}

File: demo/app/app/src/main/java/com/vectordemo/domain/dto/http/response/UserPasswordUpdateResponse.kt
package com.vectordemo.domain.dto.http.response

import com.google.gson.annotations.JsonAdapter
import com.vectordemo.utils.json.WireUserIdJsonDeserializer

data class UserPasswordUpdateResponse(
    @JsonAdapter(WireUserIdJsonDeserializer::class)
    val userId: String? = null,
    val updated: Boolean? = null,
    val message: String? = null
)
